packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx

```diff
@@ -157,21 +171,14 @@ export const SynapseChat: React.FunctionComponent<SynapseChatProps> = ({
           }}
         >
           {sessionHistory &&
-            sessionHistory.pages.map(sessionHistoryResponse => {
-              {
-                /* Note: session history is ordered from most recent to least recent in each page, so reverse the order when restoring the chat interface */
-              }
-              return sessionHistoryResponse.page
-                .reverse()
```

Comment:
Reverse is in-place, so the history order was reversed on each render

Reply:
Oops!